util/loopback.go

```diff
@@ -309,10 +309,10 @@ func getListener(address string) (listener *net.Listener, serverAddress string,
 
 	if match == "" { // Case: No given port provided for localhost
 		// Creating a listener on the next available port
-		l, err = net.Listen("tcp", ":0")
+		l, err = net.Listen("tcp", "localhost:0")
```

Comment:
Good observation. From the documentation it looks like they do not recommend having a hostname. The documentation can be found here: https://pkg.go.dev/net#Listener. In short, this is what it says:
```
The address can use a host name, but this is not recommended, because it will create a listener for at most one of the host's IP addresses.
```

Reply:
Thanks for the suggestions. I reviewed the official documentation and feel like the recommendation does not explicitly discourage localhost, but rather discourage a hard-coded fqdn, which may vary depending on where the code is deployed. So this should not be an issue for our use-case.